log/log.go

```diff
@@ -0,0 +1,214 @@
+package log
+
+import (
+	"github.com/getsentry/raven-go"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+var (
+	logger = zap.NewNop().Sugar()
+)
+
+// Level is the verbose representation of log level
+type Level string
+
+// Enums for Level
+const (
+	NopLevel   Level = "nop"
+	DebugLevel Level = "debug"
+	InfoLevel  Level = "info"
+	WarnLevel  Level = "warn"
+	ErrorLevel Level = "error"
+	PanicLevel Level = "panic"
+	FatalLevel Level = "fatal"
+
+	// This will have the same effect as nop but slower
+	ZapNopLevel zapcore.Level = zapcore.FatalLevel + 1
+)
+
+// InitLogger provides a quick way to start or replace a logger
+// Pass in debug as log level enables development mode (which makes DPanicLevel logs panic).
+func InitLogger(logLevel Level) {
+	var config = zap.NewProductionConfig()
```

Comment:
style nit: Prefer `config := ...` over `var config = ...`. (https://github.com/uber-go/guide/blob/master/style.md#local-variable-declarations)

Reply:
thanks for the link! I updated the code.